src/DrawManager.sol

```diff
@@ -343,6 +348,24 @@ contract DrawManager {
 
   /* ============ State Functions ============ */
 
+  function computeRewards(UD2x18[] memory _rewardFractions, uint256 _reserve) external pure returns (uint256[] memory rewardAmounts) {
+    (rewardAmounts,) = RewardLib.rewards(_rewardFractions, _reserve);
+  }
```

Comment:
We should probably add another `computeRewards` function that calculates the current rewards without needing any input to make it easier for bots to compute.